### DISCUSSION TITLE
Pagination not show

### DISCUSSION THREAD

**Discussion by xeno024** [bootscore/bootscore#416] [Q&A]
Hello!
My question is why the "pagination" does not appear anywhere on the page. I tried an external plugin but it doesn't show up either. As soon as I change the topic to another one it appears. What should I set to make it appear?
Thanks in advance!

**Comment by crftwrk** [comment]
Do you mean the pagination in the loop? Amount of posts is set to 24. So, you need at least 25 posts to show pagination. 

Use function to set amount in child as described here https://bootscore.me/documentation/functions/#Example_Pagination_amount_of_posts and set amount as you want to show pagination.

Does this help?

**Reply by xeno024** [reply]
Yes, that way. I was also looking at this page, I use this template with woocommerce plugin, I'm working on a tiny webshop, but I can't get it to display. I have already changed the numbers to 5, but still the pagination does not appear anywhere on the page. But as I wrote, other plugins don't display it either (e.g.: I tried WP-PageNavi, but it doesn't work either). It's like there is no place on the page, I mean in the HTML..

**Reply by crftwrk** [reply]
A link to your site will be very helpfull ;-)

**Reply by xeno024** [reply]
This is the test page, with test products
https://xeno92.ddns.net/wpxeno/index.php/termekek/

**Reply by crftwrk** [reply]
You are not using the [bootcommerce-child](https://bootscore.me/documentation/woocommerce/#bootCommerce_Child) theme. You are using the main-theme where the WooCommerce support is deactivated, that's why there are missing functions.

If you want to work on main-theme instead using a child, you must activate WooCommerce there following this [steps](https://bootscore.me/documentation/woocommerce/#Activate_WooCommerce_in_main-theme). But high risky ;-)

**Reply by xeno024** [reply]
Well thank you very much! Then that was the mistake. It's working!
Thank you very much!

**Reply by crftwrk** [reply]
You‘re welcome 👍 Please mark the answer as solved, thank you.
